tips/TIP-0027/tip-0027.md

```diff
@@ -0,0 +1,318 @@
+---
+tip: 27
+title: IOTA NFT Standard IRC27
+description: Define NFT metadata standard, policy registry and creator royalties 
+author: Adam Eunson (@AdamCroply) <adam@tanglelabs.io>, Merul Dhiman (@coodos) <merul@tanglelabs.io>
+discussions-to: https://github.com/iotaledger/tips/discussions/59
+status: Draft
+type: Standards
+layer: IRC
+created: 2022-03-04
+---
+
+
+# iNFT Standard - IRC27
```

Comment:
tbh I'm not a big fun of the iNFT prefix. It wasn't even obvious to me that it's for IOTA....It's confusing. Why don't just call it NFT Standard - IRC27 ?

Reply:
I love it! 💯